app/config/av-scan.js

```diff
@@ -0,0 +1,35 @@
+const Joi = require('joi')
+const { DEVELOPMENT } = require('../constants/enviroments')
+
+const schema = Joi.object().keys({
+  tokenUrl: Joi.string().uri().required(),
+  clientId: Joi.string().required(),
+  clientSecret: Joi.string().required(),
+  scope: Joi.string().required(),
+  grantType: Joi.string().required(),
+  avBaseUrl: Joi.string().uri().required(),
+  connectionStr: Joi.string().required(),
+  container: Joi.string().default('av-container'),
+  folder: Joi.string().default('av-files')
+})
+
+const config = {
+  tokenUrl: process.env.AV_ACCESS_TOKEN_URL,
+  clientId: process.env.AV_CLIENT_ID,
+  clientSecret: process.env.AV_CLIENT_SECRET,
+  scope: process.env.AV_SCOPE,
+  grantType: process.env.AV_AUTH_GRANT_TYPE,
+  avBaseUrl: process.env.AV_BASE_URL,
+  connectionStr: process.env.AZURE_STORAGE_CONNECTION_STRING,
+  container: process.env.AZURE_STORAGE_CONTAINER_NAME,
+  folder: process.env.AZURE_STORAGE_FOLDER_NAME
```

Comment:
You've already confirmed that the env vars referenced on lines 17-25 (excluding `AZURE_STORAGE_CONNECTION_STRING` and `AZURE_STORAGE_ACCOUNT_NAME`) are for the time being just for reference, but in general once you've implemented anything with new env vars, after merging to main just share out those variables with the team so that everyone's `.env` files are all up to date. 

I'm thinking it might be worth us as a team creating a Confluence page with the names of all the environment variables needed to run the service locally (with out the actual values of course so we're not exposing secrets), something along the lines of :
```
MESSAGE_HOST=
MESSAGE_USER=
MESSAGE_PASSWORD=
TOPIC_ADDRESS_1=
SUBSCRIPTION_ADDRESS_1=
TOPIC_ADDRESS_2=
SUBSCRIPTION_ADDRESS_2=
.
.
.
etc.
```
That way all of the devs can view the document from time to time, compare it to their own local `.env` file and if they're missing any variables can ask via chat for the actual values? 🤔 Will bring this up to the team on Monday and see what people's thoughts are.